CHANGELOG.md

```diff
@@ -852,6 +852,7 @@
 - [Add method call info for infix operators][7090]
 - [`executionComplete` response is sent on successful execution only][7143]
 - [Send info about function values][7168]
+- [Added NodeJS based tool to watch for file changes][7208]
```

Comment:
1. I believe this is not part of Enso Compiler.
2. Sentences should have dot at the end (I see other logs don't have it either - to be fixed by our Engine team, CC @JaroslavTulach).

Reply:
Moved to IDE even though it also feels weird there.